### PR TITLE
[DOC] Serve apidocs at GET /

### PR DESCRIPTION
**Pull request by stevenzeiler**
Almost all files in this commit are auto-generated by `apidoc`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,10 @@ HTTP/JSON endpoint documentation is maintained in source control using [ApiDoc](
 ApiDoc annotations exist above individual controller actions in `api/controllers/`
 
 ````
-npm run apidoc
+npm run apidoc:build
 ````
 
-Will generate documentation and serve it at [http://127.0.0.1:3000](http://127.0.0.1:3000)
+Will generate documentation in the /assets directory
```
Comment (wilsonianb) [inline README.md]: Should this say `update documentation` since it already exists?

**Comment by stevenzeiler** [comment]
:+1:

**Comment by wilsonianb** [comment]
Is it weird to show `https://api.validators.ripple.com` when running locally or staging?

**Comment by wilsonianb** [comment]
What if we changed the url field in apidoc.json to be empty?

**Comment by stevenzeiler** [comment]
I like the URL with `https://api.validators.ripple.com`, it is the only one that actually matters to people.

**Comment by wilsonianb** [comment]
LGTM :+1:

**Pull request merged**